app/controllers/think_feel_do_dashboard/moderators_controller.rb

```diff
@@ -9,8 +9,13 @@ class ModeratorsController < ApplicationController
     # POST /coach/groups/:group_id/moderates
     def create
       authorize! :moderate, @group
-      sign_in @group.moderating_participant
-      redirect_to social_networking.social_networking_profile_path
+      if @group.active_memberships.empty?
+        redirect_to root_path, alert: "Could not log in as moderator "\
+        "because there are no active membership for this group."
```

Comment:
This text sounds a little strange, is it "because there are no active membership**s** for this group." or "because there **is** no active membership for this group."